kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigWriter.java

```diff
@@ -490,12 +500,8 @@ private String debianWriteUtility(NetInterfaceConfig<? extends NetInterfaceAddre
                                         .append("\n");
                             }
                         } else {
-                            // DEFROUTE
-                            if (((NetConfigIP4) netConfig).getStatus() == NetInterfaceStatus.netIPv4StatusEnabledLAN) {
-                                sb.append("\tpost-up route del default dev ");
-                                sb.append(interfaceName);
-                                sb.append("\n");
-                            }
+                            // 'Manual' configuration
+                            sb.append("manual\n");
```

Comment:
Same here for Debian interface configuration file. Isn't this interface configured by someone else? Are we overwriting existing configuration?